producers-api/build.gradle

```diff
@@ -1,39 +1,31 @@
-apply plugin: 'info.solidsoft.pitest'
-
 sourceSets.main.java.srcDirs += "src-gen/main/java"
 
-jar {
-  enabled = true
-  archiveBaseName = 'civil-sdt-producers-api'
+sourceSets {
+    testUnit {
+        compileClasspath += sourceSets.main.output
+        runtimeClasspath += sourceSets.main.output
+        java {
+            srcDir file('src/unit-test/java')
+        }
+        resources.srcDir file('src/unit-test/resources')
+    }
 }
 
-dependencyCheck {
-    suppressionFile = "${rootDir}/config/owasp/suppressions.xml"
+configurations {
+    testOutput
+    testOutput.extendsFrom(testUnitImplementation)
+    wsdl2java
 }
 
 dependencies {
     implementation project(':domain')
     implementation project(':utils')
-    testImplementation project(":domain").sourceSets.unitTest.output
-    testImplementation project(":utils").sourceSets.unitTest.output
-
-    testImplementation project(":domain").sourceSets.integTest.output
-    testImplementation project(":utils").sourceSets.integTest.output
-}
-
-bootJar {
-    enabled = false
-}
 
-test {
-    useJUnitPlatform()
-}
+    testImplementation project(path: ':domain', configuration: 'testOutput')
```

Comment:
Is this testImplementation line needed?  The producers-api module only contains XSD validation tests so it may not need anything from the domain module tests.

Reply:
removed it now

Reply:
Confirmed